woocommerce-gateway-ebanx/gateways/class-wc-ebanx-credit-card-gateway.php

```diff
@@ -326,15 +326,22 @@ public function process_payment($order_id)
 	public function fetch_acquirer_max_installments_for_price($price, $country = null) {
 		$max_instalments = WC_EBANX_Constants::MAX_INSTALMENTS;
 		$country = $country ?: WC()->customer->get_country();
+		$currency_code = get_woocommerce_currency();
```

Comment:
We already have a property on gateway root `$this->merchant_currency = strtoupper(get_woocommerce_currency());`. Please, use it rather than that.